### ISSUE TITLE
Resolve address record inconsistencies

### ISSUE THREAD

**Issue by jannasmith** [CityOfLosAngeles/data-workflows-101#16]
Addresses and intersections entered into records from written reports use inconsistent naming conventions, such as spelling "Avenue" instead of writing "Ave." or abbreviating "MLK Blvd" instead of spelling out "Martin Luther King Jr Blvd", creating separate categories for records that should be combined into the same record.

**Comment by igotcharts** [comment]
I think this can be solved using a fuzzy matching algorithm. Do we have a master list of "true" street names?

How incorrect are some of these names? Is it just a matter of "ave" vs" avenue" or is it possible we would see something like "Maryin Lugher King Street"?

**Comment by jannasmith** [comment]
Yes we can designate a master list of official street names. Very rarely do we have major misspellings, but there are some common mistakes that we can identify
